src/main/java/org/gridsuite/modification/server/entities/equipment/deletion/ByFilterDeletionEntity.java

```diff
@@ -0,0 +1,76 @@
+/*
+  Copyright (c) 2023, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.equipment.deletion;
+
+import com.powsybl.iidm.network.IdentifiableType;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.gridsuite.modification.server.dto.ByFilterDeletionInfos;
+import org.gridsuite.modification.server.dto.FilterInfos;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+import org.gridsuite.modification.server.entities.equipment.modification.VariationFilterEntity;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "byFilterDeletion")
+@PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "by_filter_deletion_id_fk_constraint"))
+public class ByFilterDeletionEntity extends ModificationEntity {
+    @Enumerated(EnumType.STRING)
+    @Column(name = "equipmentType")
+    private IdentifiableType equipmentType;
+
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    @JoinTable(
```

Comment:
@SlimaneAmar, I reused the variation_filter table which was used with a join table for load/generator scaling. I'm not sure why we used a join table instead of a join column for this. What's your opinion on this? Do you recommend to avoid the use of join table for performance improvement (because it avoids the joining of an extra table)?

Reply:
As discussed, we will use join table as filters are used by multiple modifications. It avoid a multiple join column in the filter table for the multiple modification types.